### DISCUSSION TITLE
Add a "description" and "comments" field for Projects

### DISCUSSION THREAD

**Discussion by astrohart** [community/community#83968] [Projects and Issues] [answered]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

I am new to `Project`s, but it seems as if it could be a highly useful feature.  When I went to create a new `Project` just now, it let me give the new `Project` a title, but would not allow me to enter a more detailed description for the project.  I'd like to be able to have the ability to put a detailed description, or a `README.md` or something, attached to the project so I can go into a deeper dive as to why the project was created and what its objectives are.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by daniel-evers** [comment, marked as answer]
This is already possible. See https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/best-practices-for-projects#make-use-of-the-description-and-readme

**Reply by astrohart** [reply]
Yes, upon further investigation, I did find that.

I guess, however, I might not have asked the question has there been a description field on the main project screen.

That would be my suggestion, is to put a field there, so I don't have to go hunting for it.

Thank you!
